client/ionic/android/app/src/main/AndroidManifest.xml

```diff
@@ -46,19 +46,8 @@
     <!-- Permissions -->
 
     <uses-permission android:name="android.permission.INTERNET" />
-    <!-- Camera, Photos, input file -->
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
     <!-- Geolocation API -->
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-feature android:name="android.hardware.location.gps" />
-    <!-- Network API -->
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
Why don't we need ACCESS_NETWORK_STATE?

Reply:
Also, for the first version, do we actually need access location?

Reply:
@epicfaace `ACCESS_NETWORK_STATE` allows you to control whether to access the internet only when connected to WiFi or sth like that. Typically not used for foreground operation, and v0.1 app is all foreground

@myiremark I believe the answer at the standup meeting was "no"

Reply:
My understanding of ACCESS_NETWORK_STATE is that it is needed if you want to check if you are connected to a network or not, i.e. not likely needed at present.  The Ionic code I looked at today only seems to use it in the [Network API](https://github.com/ionic-team/capacitor/blob/master/site/docs-md/apis/network/index.md).  The other reference I found was in the [(Ionic) Capacitor Android Plugin Guide
](https://github.com/ionic-team/capacitor/blob/master/site/docs-md/plugins/android.md).

I'll comment out the location access for now in another commit, coming shortly.

Reply:
The location access permissions have been commented out for now.